demo/3000-home/components/SharedNav.js

```diff
@@ -28,26 +37,37 @@ const SharedNav = () => {
           {
             label: (
               <>
+                <Badge color={badgeColor(homeRemote)} />
                 Home <sup>3000</sup>
               </>
             ),
             key: '/',
+            onMouseEnter: () => {
+              // prefetch remote container on HOVER manually
+              // if you use `next/link` it prefetches remoteEntry automatically
+              // but here Antd.Menu does not use Link, so do it manually
+              homeRemote.remote.getContainer();
+            },
           },
           {
             label: (
               <>
+                <Badge color={badgeColor(shopRemote)} />
                 Shop <sup>3001</sup>
               </>
             ),
             key: '/shop',
+            onMouseEnter: () => shopRemote.remote.getContainer(),
           },
           {
             label: (
               <>
+                <Badge color={badgeColor(checkoutRemote)} />
                 Checkout <sup>3002</sup>
               </>
             ),
             key: '/checkout',
+            onMouseEnter: () => checkoutRemote.remote.getContainer(),
```

Comment:
if next link were used, does federated route get fetched automatically?

Reply:
In PROD remoteEntry & exposed module even be prefetched automatically on mouse hover.
In DEV only by next/link click, because in dev prefetch is disabled.

Reply:
gotcha!